pkg/networkgraph/tree/wrapper.go

```diff
@@ -49,6 +51,10 @@ func NewNetworkTreeWrapper(entities []*storage.NetworkEntityInfo) (NetworkTree,
 	}, nil
 }
 
+func (t *networkTreeWrapper) findCIDRNoLock(ipNet *net.IPNet) (*nRadixNode, error) {
```

Comment:
This was needed after adding `findCIDRNoLock` to the `NetworkTree` interface. Should it be implemented?

Reply:
Removed